bin/summarize-user-events

```diff
@@ -146,18 +147,18 @@ class SummarizeUserEvents
   end
 
   def find_cloudwatch_events(&block)
-    if $stdin.tty?
+    unless file_name.nil?
+      warn 'Reading Cloudwatch events as newline-delimited JSON (ndjson) a file'
+      file_source(&block)
+    else 
       cloudwatch_source(&block)
-    else
-      warn 'Reading Cloudwatch events as newline-delimited JSON (ndjson) from stdin'
-      stdin_source(&block)
     end
   end
 
-  def stdin_source(&block)
+  def file_source(&block)
     events = []
 
-    $stdin.each_line do |line|
+    File.read(file_name).each_line do |line|
```

Comment:
A common pattern in shell tools is to allow "-" to mean "use stdin". So it would be cool to just check if `file_name == '-'` and use stdin in that case (so, keep stdin_source and update `find_cloudwatch_events` to check if '-' has been passed as a filename and use stdin_source in that case, otherwise use file_source).

Reply:
I am going to merge the PR as is and make a note to come back to this after the holidays.